screens/MatchesDashboard.tsx

```
@@ -3,99 +1,179 @@
                     </TouchableOpacity>
-                </View>
-            ))}
+                    {showMatches && (
+                        <ThemedView style={commonStyles.section}>
+                            {dater.matchRecommendations.map((matchRecommendation: MatchRecommendation) => {
+                                const match = getDaterById(matchRecommendation.daterId);
+                                if (!match){
+                                    return (
+                                        <ThemedText style={commonStyles.errorText}>
+                                            Match not found
+                                        </ThemedText>
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)